documentation/user_docs/docs/manual/Manipulating_and_extracting_data_from_SQW_files_and_objects.rst

```diff
@@ -12,13 +12,13 @@ There are various different forms of input for this function, the purpose of whi
    my_cut = cut_sqw (data_source, proj, p1_bin, p2_bin, p3_bin)
 
 
-``data_source`` is a string giving the full filename (including path) of the input SQW file.
+Where ``data_source`` is a string giving the full filename (including path) of the input SQW file or just SQW object stored in memory.
 
-**Projection axes**
+** Projection **
 
 This defines the coordinate system you will use to view the data.
 
-:ref:``proj`` is a Matlab structure array containing information about the axes you wish to use to view the data. Because each point in the SQW file is labelled with h, k, and l (the reciprocal lattice vectors), and energy, it is possible, if you wish, to redefine the co-ordinate system with one of your choosing. For example you may wish to view the data in terms of (h,h,0)/(h,-h,0)/(0,0,l). This is distinct from the vectors ``u`` and ``v`` that are specified in `gen_sqw <List_of_functions:gen_sqw>`, which describe how the crystal is oriented with respect to the spectrometer and are determined by the physical orientation of your sample.
+:ref:``proj`` the instance of ``ortho_proj`` class containing information about the axes and the coordinate system you wish to use to view the data. You can also use the structure, with the same fields as the ``ortho_proj`` class, but using the class is usually easier as the class properties are Because each point in the SQW file is labelled with h, k, and l (the reciprocal lattice vectors), and energy, it is possible, if you wish, to redefine the co-ordinate system with one of your choosing. For example you may wish to view the data in terms of (h,h,0)/(h,-h,0)/(0,0,l). This is distinct from the vectors ``u`` and ``v`` that are specified in `gen_sqw <List_of_functions:gen_sqw>`, which describe how the crystal is oriented with respect to the spectrometer and are determined by the physical orientation of your sample.
```

Comment:
``proj`` IS AN instance of ``ortho_proj`` class......

No comma between structure and with the same fields, and prefer A structure to THE structure.

Sentence before "Because each point" is incomplete. The class properties are something but you don't say what.